eat_places.md

```diff
@@ -0,0 +1,3 @@
+## places to feast
```

Comment:
theres a good one in El Cerrito on San Pablo next to Hotsy Totsy... i usually get al pastor